integration_tests/test_cargo_audit.py

```diff
@@ -0,0 +1,12 @@
+from subprocess import Popen, PIPE, STDOUT
+
+
+def test_cargo_audit():
+
+    cmd1 = 'cargo build --all-features --release'
+    cmd2 = 'cargo-audit audit'
+    prs = Popen("{}; {}".format(cmd1, cmd2), shell=True, stdin=PIPE,
+            stdout=PIPE, stderr=STDOUT, close_fds=True)
+    stdout, nothing = prs.communicate()
```

Comment:
Thanks for the PR @anth1y . Any reason why we can't just run `cargo audit` on the repo?

Reply:
I think I am the culprit :)) because of this: https://github.com/rust-vmm/rust-vmm-ci/pull/36#discussion_r484308058

I was running `cargo audit` at some point and it would fail if `Cargo.lock` was not in the repository. I doubled checked now and it looks like if Cargo.lock does not exist, it will be created when running `cargo audit`, so all good. We do not need to run `cargo build` before. Good catch, @sameo!